components/invitation_modal/invitation_modal_members_step/index.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {connect} from 'react-redux';
+
+import {
+    getConfig,
+    getLicense,
+} from 'mattermost-redux/selectors/entities/general';
+import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';
+
+import {GlobalState} from 'types/store';
+
+import {isAdmin} from 'utils/utils.jsx';
+
+import InvitationModalMembersStep from './invitation_modal_members_step';
+
+function mapStateToProps(state: GlobalState) {
+    return {
+        userLimit: getConfig(state).ExperimentalCloudUserLimit,
+        currentUsers: state.entities.admin.analytics!.TOTAL_USERS,
```

Comment:
I see you have a null check there but what happens if the analytics is not yet in redux? 

We should call [`getStandardAnalytics`](https://github.com/mattermost/mattermost-redux/blob/master/src/actions/admin.ts#L432) if `state.entities.admin.analytics` when the component mounts to guarantee that we are always checking against a correct current user count.